lua/possession/commands.lua

```diff
@@ -174,6 +188,11 @@ function M.list(full)
     display.echo_sessions { vimscript = full }
 end
 
+---@param full? boolean
+function M.list_cwd(full)
+    display.echo_sessions { vimscript = full, only_cwd = true }
```

Comment:
(regarding comment in `display.lua`) This would be something like 
```lua
display.echo_sessions {
    vimscript = full,
    sessions = query.filter(query.as_list(), { cwd = true }) 
}
```
This is more verbose, but I believe there is clear separation of functionality between display/query modules.

Reply:
Change made.